platforms/netlify-zisi/README.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+Netlify's current build bot uses an older version of zip-it-and-ship-it. This is however, overrideable using an env var called `ZISI_VERSION`.
```

Comment:
What is "build bot"?
Link `zip-it-and-ship-it` to https://github.com/netlify/zip-it-and-ship-it

Reply:
They call their CI build `build bot`.

Reply:
Changed it to build CI.